Index: settting.gradle
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/settting.gradle b/settting.gradle
new file mode 100644
--- /dev/null	(date 1776661670655)
+++ b/settting.gradle	(date 1776661670655)
@@ -0,0 +1,1 @@
+rootProject.name = 'SportStore'
\ No newline at end of file
Index: build.gradle
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/build.gradle b/build.gradle
new file mode 100644
--- /dev/null	(date 1778180072969)
+++ b/build.gradle	(date 1778180072969)
@@ -0,0 +1,99 @@
+plugins {
+    id 'java'
+    id 'war'
+    id "org.hidetake.ssh" version "2.11.2"
+}
+
+group = 'com.shop'
+version = '1.0-SNAPSHOT'
+
+java {
+    toolchain {
+        languageVersion = JavaLanguageVersion.of(21)
+    }
+}
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+
+    compileOnly 'jakarta.servlet:jakarta.servlet-api:6.0.0'
+    implementation 'org.apache.tomcat:tomcat-jasper:10.1.52'
+    implementation 'jakarta.servlet.jsp.jstl:jakarta.servlet.jsp.jstl-api:3.0.1'
+    implementation 'org.glassfish.web:jakarta.servlet.jsp.jstl:3.0.1'
+    implementation 'com.google.code.gson:gson:2.10.1'
+    implementation 'com.mysql:mysql-connector-j:8.3.0'
+
+    compileOnly 'org.projectlombok:lombok:1.18.30'
+    annotationProcessor 'org.projectlombok:lombok:1.18.30'
+
+    testImplementation 'org.junit.jupiter:junit-jupiter:5.13.2'
+}
+
+tasks.test {
+    useJUnitPlatform()
+}
+
+// ================= SSH =================
+
+apply plugin: 'org.hidetake.ssh'
+
+
+remotes {
+    server {
+        host = '192.168.30.130'
+        port = 22
+        user = 'huynhtran'
+        password = '123456'
+    }
+}
+
+
+ssh.settings {
+    knownHosts = allowAnyHosts
+}
+
+// ================= TASK =================
+task uploadWar {
+    dependsOn build
+
+    doLast {
+        ssh.run {
+            session(remotes.server) {
+
+                execute 'mkdir -p /usr/deploy'
+
+                put from: file("build/libs/${project.name}-${version}.war"),
+                        into: "/usr/deploy/ROOT.war"
+            }
+        }
+    }
+}
+
+task restartDocker {
+    doLast {
+        ssh.run {
+            session(remotes.server) {
+
+                // Xóa container cũ
+                execute 'docker rm -f tomcat10 || true'
+
+                // Chạy container mới
+                execute '''
+docker run -d \
+--name tomcat10 \
+--network sport-network \
+-v /usr/deploy:/usr/local/tomcat/webapps \
+-p 80:8080 \
+tomcat:10.1
+'''
+            }
+        }
+    }
+}
+
+task deploy {
+    dependsOn uploadWar, restartDocker
+}
\ No newline at end of file
Index: .idea/gradle.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/gradle.xml b/.idea/gradle.xml
new file mode 100644
--- /dev/null	(date 1776669406280)
+++ b/.idea/gradle.xml	(date 1776669406280)
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="GradleMigrationSettings" migrationVersion="1" />
+  <component name="GradleSettings">
+    <option name="linkedExternalProjectsSettings">
+      <GradleProjectSettings>
+        <option name="externalProjectPath" value="$PROJECT_DIR$" />
+        <option name="modules">
+          <set>
+            <option value="$PROJECT_DIR$" />
+          </set>
+        </option>
+      </GradleProjectSettings>
+    </option>
+  </component>
+</project>
\ No newline at end of file
